colossalai/inference/requirements.txt

```diff
@@ -0,0 +1,3 @@
+transformers==4.36.2
+ordered_set
+gpustat
```

Comment:
we do not need this file for reasons below:
1. we never put a requirement file inside a module, requirement file should in the setup workspace.
2. we don't need gpustat for running this module
3. try to replace ordered_set with list to reduce dependency on third-party lib

you can fix the transformers version in `requirements/requirements.txt` in the project root directory.

Reply:
Gotcha gotcha. I'll make this PR only remove unused files and add and revise requirements after merging the other PRs and finalizing the version. `ordered_set` is expected to be replaced in another WIP PR.

Reply:
sounds good to me